client/state/themes/actions/add-external-managed-theme-to-cart.tsx

```diff
@@ -36,7 +36,7 @@ const isLoadingCart = ( isLoading: boolean ) => ( dispatch: CalypsoDispatch ) =>
  * @param products list of products
  * @returns string
  */
-function getPreferredBillingCycleProductSlug(
+export function getPreferredBillingCycleProductSlug(
```

Comment:
Feels weird that we're exporting a non-action from an actions file. Maybe we should move this function somewhere else?